tests/sphere_test.cpp

```diff
@@ -0,0 +1,120 @@
+#include "obj/objects.h"
+#include "gtest/gtest.h"
+
+using core::Ray;
+using core::RayHitData;
+using core::Vec3;
+using objects::Sphere;
+
+const float kSkipFreq = 1000;
+const Vec3 kVecZero(0, 0, 0);
```

Comment:
Code reuse.

Reply:
Why you do not want to put it in a `constants` namespace?

How about just moving this vec to Vec3 class? E.g

```c++
class Vec3 {
 public:
  static const Vec3 kZero(0, 0, 0);
};
```

kSkipFreq is ok to keep here, as there is no good place to put it. It is not as generic value as zero vector.

Reply:
Beause for me this not belongs there. Vec3(0, 0, 0) is not a constant that is required for simulation, whereas others are. 

Putting this into Vec3 as you suggested is fine ;)

Reply:
I can't do this. 
![image](https://user-images.githubusercontent.com/45213495/99542526-79d74280-29b2-11eb-80b5-4183ee23f367.png)

should I do this instead? 
![image](https://user-images.githubusercontent.com/45213495/99542874-d175ae00-29b2-11eb-9431-47c24fb58d82.png)

Reply:
this is the error that im getting btw: 
![image](https://user-images.githubusercontent.com/45213495/99542999-f538f400-29b2-11eb-9197-e3554c4133e8.png)

Reply:
I did this: 
![image](https://user-images.githubusercontent.com/45213495/99544096-341b7980-29b4-11eb-9ff0-1391c496cab4.png) and refractored all tests ;)